File: Modul I/Bahasa C/PRAK110_2310817220028_SheilaSabina.c
#include <stdio.h>

void main(){
    int x = 5;
    int y = 12;
    int z = pow (y,2) + pow (x,2);
    int a = sqrt(z);
    int b = (x+y+a);
    int c = (x*y)/2;

    printf("Diketahui:\n");
    printf("Alas = %d cm\n", x);
    printf("Tinggi = %d cm\n\n", y);

    printf("Jawab:\n");
    printf("Sisi A = %d cm\n", y);
    printf("Sisi B = %d cm\n", a);
    printf("Sisi C = %d cm\n", x);
    printf("Keliling = %d cm\n", b);
    printf("Luas =  %d cm\n", c);
}

File: Modul I/Bahasa Python/PRAK110_2310817220028_SheilaSabina.py
import math

x = 5
y = 12
z = int(math.sqrt(y**2 + x**2))
a = x + y + z
b = int(1/2*x*y)

print(f"Diketahui :\nAlas = {x} cm\nTinggi = {y} cm\n")
print(f"Jawab : \nSisi A = {y} cm\nSisi B = {z} cm\nSisi C = {x} cm")
print(f"Keliling = {a} cm\nLuas = {b} cm²")